User microservice/database_setup.py

```diff
@@ -0,0 +1,42 @@
+import configparser
+from sqlalchemy import create_engine
+
+from sqlalchemy.orm import declarative_base, sessionmaker
+
+config = configparser.ConfigParser()
+config.read('config.ini')
+
+db_username = config['DATABASE']['Username']
+db_password = config['DATABASE']['Password']
+db_host = config['DATABASE']['Host']
+db_name = config['DATABASE']['Name']
+
+db_username = config['DATABASE']['Username']
+db_password = config['DATABASE']['Password']
+db_host = config['DATABASE']['Host']
+db_name = config['DATABASE']['Name']
```

Comment:
I think you need to delete this, this is duplicate code

Reply:
Why there are duplicated ?

Reply:
you are repeated them 😂 

take a look at line 9